assembl/graphql/vote_session.py

```diff
@@ -266,6 +294,16 @@ def mutate(root, args, context, info):
                 instructions=instructions_ls,
                 exclusive_categories=exclusive_categories
             )
+            proposal_id = args.get('proposal_id')
+            if proposal_id:
+                proposal_id = int(Node.from_global_id(proposal_id)[1])
+                proposal = models.Idea.get(proposal_id)
+                vote_spec.criterion_idea = proposal
+
+            vote_spec_template_id = args.get('vote_spec_template_id')
+            if vote_spec_template_id:
+                vote_spec.settings = json.dumps({'vote_spec_template_id': vote_spec_template_id})
```

Comment:
This can be a bit unsafe. Verify first that it is a JSON-type before you putting this blob into the db.

Reply:
Also, why put Relay style id's into the settings instead of database id's?

Reply:
I don't see what is unsafe here. The vote_spec_template_id here is already validated by graphene, it's an ID type, so basically a string.

I should probably use a vote_spec_template_id foreign key, yes, I was just lazy yesterday. I need to write another alembic step, sigh.

Reply:
Ok, it's now a forieign key